src/resolvers/mutation/create-deployment/index.js

```
@@ -99,10 +99,24 @@ export default async function createDeployment(_, args, ctx, info) {
     ? validateReleaseName(args.releaseName)
     : generateReleaseName();
 
-  // Add default props if exists
+  // Determine if proeprties.extra_au exists, if not, default to
+  // executor config defaultExtraAu
+  const providedExtraAU = get(
+    args,
+    "properties.extra_au",
+    executorConfig.defaultExtraAu
+  );
+
+  // Validate extra au amount is at least the default
+  const extraAu =
+    providedExtraAU < executorConfig.defaultExtraAu
+      ? executorConfig.defaultExtraAu
+      : providedExtraAU;
```

@samblackk nice! I guess this raises another question. Should we auto-bump them here, like we have, or should we throw a validation error like we do for other invalid API requests? I know our UI and CLI won't have this issue, but just wondering in the case of an outside request.

---

@mrrobby @andriisoldatenko thoughts? Should we throw validation error or simply auto-set the correct extraAu if none is provided?

---

i prefer to fallback to default value, instead of raise error.

---

Maybe fallback is fine for now. Once the schema has been cleaned up a bit more, we could look at documenting and validating the input more precisely.